### ISSUE TITLE
Jupyter no longer displays equations

### ISSUE THREAD

**Issue by arkajitmandal** [microsoft/vscode-jupyter#4877]
## Environment data

-   VS Code version: 1.53.2
-   Jupyter Extension version (available under the Extensions sidebar): v2021.2.576440691
-   Python Extension version (available under the Extensions sidebar): 3.8.3
-   OS (Windows | Mac | Linux distro) and version: Mac

## Expected behaviour

Any equations typed within '$' or between 'align' tags used to work.  

## Actual behaviour

No equations show up. Instead the raw text is shown. 

<img width="1392" alt="Screen Shot 2021-02-18 at 11 01 30 PM" src="https://user-images.githubusercontent.com/10487689/108456306-060d3600-723e-11eb-802c-6caf973a575a.png">
<img width="1392" alt="Screen Shot 2021-02-18 at 11 01 46 PM" src="https://user-images.githubusercontent.com/10487689/108456321-0a395380-723e-11eb-8068-493bccb0fa1c.png">

**Comment by claudiaregio** [comment]
Hi @arkajitmandal , thank you for the feedback! The reason that this is happening is because you have been randomly selected to get a preview of our [new notebook experience](https://github.com/microsoft/vscode-jupyter/wiki/Introducing-Native-Notebooks) inside VS Code. One of the limitations we currently have is that Latex or other math markdown is not yet supported. Updates on the support for this feature can be found here: https://github.com/microsoft/vscode-jupyter/issues/1216 
We would love it if you could continue and try to use this new experience and give us additional feedback about it. But if your workflow dictates that math markdown be supported, you can opt out of the preview by adding this to your user settings.json.
```json
    "jupyter.experiments.optOutFrom":[
        "NativeNotebookEditor"
    ],
```
This file is accessible by opening your settings (Ctrl-,) then clicking on the Open Settings (JSON) button. 
![image](https://user-images.githubusercontent.com/11685408/108561113-8d948c80-72b2-11eb-9242-065c1e49ce71.png)
Thank you for being a VS Code notebook user and providing this feedback!

**Comment by arkajitmandal** [comment]
Thank a lot for your response. Yes that fixes my problem for now. Overall, I do like the new layout, but for me it is difficult to live by without the Math support in the md.

**Comment by gusbemacbe** [comment]
And I am a Linux user. While, everything works only with classic and native Jupyter Notebook. :-( I hope Microsoft add support for any Jupyter Notebook extensions as Python Markdown for VSCode's Jupyter extension.

**Comment by greazer** [comment]
This will be addressed by #1216, which is on the way.
